### ISSUE TITLE
Player data deleted when loading into SP

### ISSUE THREAD

**Issue by mnewbold01** [GTNewHorizons/GT-New-Horizons-Modpack#18475]
### Your GTNH Discord Username

Drewdroid9#1341

### Your Pack Version

2.7.1

### Your Server

SP

### Java Version

Java 21

### Type of Server

Single Player

### Your Expectation

When logging in, world should load as normal with player inventory from previous session.

### The Reality

Sometimes if the world is taking a while to load, it will eventually startup but the player (me) will be in seed spawnpoint with just a questbook in inventory. The rest of the world is fine and all waypoints are still there.

I've tried replacing level.dat files & playerdata folder with the ones in a backup but nothing allows me to get my inv back. Have just been loading up a backup world every time this happens (which can get annoying obviously).

### Your Proposal

fix pls x

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by Lyfts** [comment]
Are you logging in with a different user or playing cracked?

**Comment by mnewbold01** [comment]
No, same user. Not changing anything between sessions. It won't happen all the time, just if the game is taking unusually long to load I'll find it happens. MultiMC btw

**Comment by samcoydev** [comment]
Hey there. I've also run into this a couple times on a brand new single player world. Same user each time. Not cracked. Although I used Curseforge, which I'm now seeing is not recommended. Has there been any update on this issue?

**Comment by brendanbennett** [comment]
Bumping because I've just got exactly the same issue. Prism launcher on Debian, 2.8.1, Java 25. No changes were made since last time except I think prism launcher might have updated

**Comment by PLASMAchicken** [comment]
Try loading a backup and make sure you are playing in online-mode
